lighthouse-cli/chrome-finder.js

```diff
@@ -34,15 +37,17 @@ module.exports = {
 
     execSync(
       `${LSREGISTER} -dump` +
-      ' | grep -i \'google chrome canary.app$\'' +
+      ' | grep -i \'google chrome[ canary]\{0,7\}.app$\'' +
```

Comment:
hmm, I have to double escape like `\\{0,7\\}` for this to match anything on my machine.

Also, would it be better to just use a group here to match the exact substring? Something like `' | grep -i \'google chrome\\( canary\\)\\?.app$\'' +`